l10n_it_asset_history_import/wizards/asset_history_import.py

```diff
@@ -434,15 +454,29 @@ def convert_to_asset_vals(self, asset_dict, workbook, sheet):
                     "Is it in its ISO 4217 format?"
                 )
             )
+        supplier_vat_col = get_header_by_model_and_field("res.partner", "vat").col
+        supplier_vat = asset_dict[supplier_vat_col]
+        partner = self.env["res.partner"]
+        if supplier_vat:
+            partner = partner.search([("vat", "ilike", supplier_vat)])[:1]
+            if not partner:
+                raise ValidationError(
+                    _("Could not find Supplier with VAT {}".format(supplier_vat))
```

Comment:
Since Odoo 14.0, the `_()` allows usage of placeholders: https://github.com/odoo/odoo/commit/3c3b88f6768e31417263b3828a874ca204e29514

It's good practice to use it when possible:
```suggestion
                    _("Could not find Supplier with VAT %s", supplier_vat)
```